sklearn/model_selection/_search.py

```diff
@@ -918,15 +1049,33 @@ class RandomizedSearchCV(BaseSearchCV):
 
     Attributes
     ----------
-    grid_scores_ : list of named tuples
-        Contains scores for all parameter combinations in param_grid.
-        Each entry corresponds to one parameter setting.
-        Each named tuple has the attributes:
-
-            * ``parameters``, a dict of parameter settings
-            * ``mean_validation_score``, the mean score over the
-              cross-validation folds
-            * ``cv_validation_scores``, the list of scores for each fold
+    search_results_ : dict of numpy (masked) ndarrays
+        A dict with keys as column headers and values as columns, that can be
+        imported into a pandas DataFrame.
+
+        For instance the below given table
+
+        param_kernel|param_gamma|param_degree|accuracy_score_train_split_0|...|
+        =======================================================================
+        'poly'      |      -    |      2     |            0.8             |...|
+        'poly'      |      -    |      3     |            0.7             |...|
+        'rbf'       |     0.1   |      -     |            0.8             |...|
+        'rbf'       |     0.2   |      -     |            0.9             |...|
+
+        will be represented by a search_results_ dict of :
+
+        {'param_kernel' : masked_array(data = ['poly', 'poly', 'rbf', 'rbf'],
+                                       mask = [False False False False]...)
+         'param_gamma'  : masked_array(data = [-- -- 0.1 0.2],
+                                       mask = [ True  True False False]...),
+         'param_degree' : masked_array(data = [2.0 3.0 -- --],
+                                       mask = [False False  True  True]...),
+         'accuracy_score_test_split_0' : [0.8, 0.7, 0.8, 0.9],
+         'accuracy_score_test_split_1' : [0.82, 0.5, 0.7, 0.78],
+         'accuracy_score_test_mean'    : [0.81, 0.60, 0.75, 0.82],
+         'accuracy_score_test_std'     : [0.02, 0.01, 0.03, 0.03],
+         'accuracy_score_test_rank'    : [2, 4, 3, 1],
+        }
```

Comment:
Disregard this comment. See [this comment](https://github.com/scikit-learn/scikit-learn/pull/6697/files#r61925306) instead.